docs/conf.py

```diff
@@ -30,7 +30,8 @@
 # read version number (and other metadata) from package init
 init_fn = os.path.join(here, '..', 'letsencrypt', '__init__.py')
 with codecs.open(init_fn, encoding='utf8') as fd:
-    meta = dict(re.findall(r"""__([a-z]+)__ = "([^"]+)""", fd.read()))
+    meta = dict(re.findall(r"""__(\w*)__\s*=\s*'(\S*)'""", fd.read()))
```

Comment:
this regex foo was very complicated already, I much more prefer simple solution from https://github.com/letsencrypt/letsencrypt/pull/904 and hope nobody changes the quotes symbol (in fact our dev release script would break in such case) :)